llvm/include/llvm/IR/DebugInfoMetadata.h

```diff
@@ -3045,7 +3045,10 @@ class DIExpression : public MDNode {
   std::pair<DIExpression *, const ConstantInt *>
   constantFold(const ConstantInt *CI);
 
-  DIExpression *constantFold();
+  /// Try to shorten an expression with constand math operations that can be
+  /// evaulated at compile time. Returns a new expression on success, or the old
+  /// expression if there is nothing to be reduced.
+  DIExpression *foldConstantMath();
```

Comment:
This is a small error in how I set up the PRs, the name should be `foldConstantMath` because I dont want it to be confused for `DIExpression:constantFold()`

They do different things